### PR TITLE
Make TOC text direction always match article text direction.

### PR DESCRIPTION
**Pull request by montehurd**
https://phabricator.wikimedia.org/T149066

When device lang is LTR...

# Before
The TOC was still LTR when the article was RTL:
before mov

# After
The TOC is RTL when the article is RTL:
after mov

### CODE REVIEW

**Comment by montehurd** [comment]
Oops forgot to test when device lang *is* RTL and you load a LTR article...

File: Wikipedia/Code/WMFTableOfContentsViewController.swift [hunk 1]
```diff
@@ -32,6 +32,8 @@ open class WMFTableOfContentsViewController: UIViewController,
                                                WMFTableOfContentsAnimatorDelegate {
     
     let tableOfContentsFunnel: ToCInteractionFunnel
+
+    let semanticContentAttribute: UISemanticContentAttribute
```
Comment (montehurd) [inline Wikipedia/Code/WMFTableOfContentsViewController.swift]: Considered naming this `semanticContentAttributeOverride`...

**Comment by joewalsh** [comment]
retest this please

File: Wikipedia/Code/WMFTableOfContentsViewController.swift (hunk 1 above)
Comment (amrox) [inline Wikipedia/Code/WMFTableOfContentsViewController.swift]: actually I like `semanticContentAttributeOverride`

**Pull request merged**